src/js/components/Select/doc.js

```diff
@@ -201,6 +201,16 @@ export const doc = Select => {
 };
 
 export const themeDoc = {
+  'global.hover.background': {
+    description: 'The background color when hovering.',
```

Comment:
I'm not sure it is right to say:

'The background color when hovering.'

the background is an object, and it can specify more than just color. e.g. in your example, it is also defining opacity for example.

Reply:
Updated language from 'The background color when hovering' to 'The background style when hovering'.